osf/management/commands/migrate_file_view_counts_from_keen.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import unicode_literals
+import logging
+
+import django
+django.setup()
+import json
+import requests
+import urllib
+from addons.osfstorage.models import OsfStorageFile
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from osf.models import PageCounter
+from website import settings
+
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def set_file_view_counts():
+    # get all osfstorage files which is_deleted == False, the file size in production database > 1730325
+    files = OsfStorageFile.objects.all()
+
+    # the limit of the datasets size return from keen is 400kb.
+    # A json file return 3 file counts from keen is about 223 bytes
+    # So a json file return from keen should be able to reach 4800 files, to be safe use 4500
+    keen_file_limit = 4500
```

Comment:
Note: We should use Django's `Paginator` or the batch iteration technique that we've used in migrations here: https://github.com/CenterForOpenScience/osf.io/blob/b7b8e9b78348f1fe4b22fca8e6110072dc497aa7/osf/migrations/0103_set_osf_storage_node_settings_region.py#L33-L37